pom.xml

```diff
@@ -257,7 +257,7 @@
     <version.org.sonatype.plexus.plexus-sec-dispatcher>1.3</version.org.sonatype.plexus.plexus-sec-dispatcher>
     <version.org.sonatype.sisu>2.3.0</version.org.sonatype.sisu>
     <version.org.sonatype.sisu.sisu-guice>3.2.3</version.org.sonatype.sisu.sisu-guice>
-    <version.org.springframework>5.2.5.RELEASE</version.org.springframework>
+    <version.org.springframework>5.3.4</version.org.springframework>
```

Comment:
Please use 5.2.9.RELEASE. That will fix OptaWeb Vehicle Routing and it's aligned with Spring Boot 2.3.4.RELEASE.

See for example: https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-starter/2.3.4.RELEASE.

Reply:
https://github.com/spring-projects/spring-boot/blob/v2.3.4.RELEASE/spring-boot-project/spring-boot-dependencies/build.gradle#L1671

Context: for some reason we (KIE) decided to manage part of the Spring dependencies ourselves here in kie-parent. With great power comes great responsibility :slightly_smiling_face: So whenever we upgrade `version.org.springframework.boot` we also need to upgrade `version.org.springframework` to keep the Spring framework dependencies aligned to the Spring Boot BOM as closely as possible. Otherwise one of the main goals of Spring Boot is lost. Spring Boot defines the exact versions of components that work together.